CommunicationLayer/CommunicationContainer.cpp

```diff
@@ -2,12 +2,9 @@
 #include <QUdpSocket>
 
 #include "../DataLayer/DataContainer.h"
-#include "CommunicationContainer.h"
-#include "CommDeviceControl/ConnectionController.h"
-#include "CommDeviceControl/CommDeviceManager.h"
```

Comment:
Here are the reasons why I don't think we need  UdpConnectionService/commDeviceManager/ConnectionController:

* These are there for the sole purpose of receiving data. For transmitting we still have the UDPForwardingService which will be modified to send to a server and not on local multicast.
* The reason they were created was to allow to dynamically switch between Radio and UDP input. The host telemetry has no reason to be listening to incoming UDP telemetry. 
* The controller needed to be there to activate and deactivate other services such as the UDP forwarding but now all those services are able to just be started on construction.
* Because the connection is extremely simplified I think it is fine to contain everything within the RadioCommDevice as all it is doing is opening up the port which is a logical construction item. Anything more than that I could justify extracting to another service.